src/index.html

```diff
@@ -1,19 +1,15 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>[CI T1] Neural Notes</title>
+    <title>Neural Notes</title>
 
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
 
     <link
-      href="https://fonts.googleapis.com/css?family=Raleway"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css?family=Roboto:300,400,700&display=swap"
+      href="https://fonts.googleapis.com/css?family=Raleway&family=ZCOOL+QingKe+HuangYou&&family=Roboto:300,400,700display=swap"
```

Comment:
Why do we need `family=ZCOOL+QingKe+HuangYou` ? @TomaTomoe are we using these fonts in our design?

The string `family=Roboto:300,400,700display=swap` is missing `&` between `700` and `display=swap`.

Reply:
fixed by https://github.com/DarthVanger/NeuralNotes/pull/167/commits/af032cb32dd4e363a0163f3c851279751ea54f6b